modules/cstrike/cstrike/CstrikeMain.cpp

```diff
@@ -67,13 +67,19 @@ void OnPluginsLoaded()
 	ForwardInternalCommand = MF_RegisterForward("CS_InternalCommand", ET_STOP, FP_CELL, FP_STRING, FP_DONE);
 	ForwardOnBuy           = MF_RegisterForward("CS_OnBuy"          , ET_STOP, FP_CELL, FP_CELL, FP_DONE);
 	ForwardOnBuyAttempt    = MF_RegisterForward("CS_OnBuyAttempt"   , ET_STOP, FP_CELL, FP_CELL, FP_DONE);
+}
+
+void OnServerActivate(edict_t *pEdictList, int edictCount, int clientMax)
+{
+	// Used to catch WeaponList message at map change.
+	EnableMessageHooks();
 
-	if (!ClientCommandDetour) // All CS_* forwards requires ClientCommand. Unlikely to fail.
+	if (!ClientCommandDetour) // All CS_* forwards requires ClientCommand. Unlikely to fail. 
```

Comment:
lol, It was a silly change to refresh build on Appveyor since for some reason AMBuild fails randomly.